### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by cafemoloko**
- fixed & updated links
- minor fixes

### CODE REVIEW

**Comment by Luxonis-Brandon** [comment]
Thanks!

File: source/pages/faq.rst [hunk 1]
```diff
@@ -683,15 +681,15 @@ Monocular Neural Inference fused with Stereo Depth Mode
 
 For DepthAI units with onboard cameras, this works out to the following minimum depths:
 
-- DepthAI Raspberry Pi Compute Module Edition (:ref:`BW1097 <bw1097>`) the minimum depth is **0.827** meters for full 1280x800 stereo resolution and  **0.414** meters for 640x400 stereo resolution:
+- DepthAI Raspberry Pi Compute Module Edition (`BW1097 <https://docs.luxonis.com/projects/hardware/en/latest/pages/BW1097.html>`__) the minimum depth is **0.827** meters for full 1280x800 stereo resolution and  **0.414** meters for 640x400 stereo resolution:
```
Comment (Erol444) [inline source/pages/faq.rst:684]: do we plan on "migrating" all HW pages to depthai-hardware repo and this is the first step?

**Review by Erol444 — APPROVED** [review]

File: source/pages/faq.rst (hunk 1 above)
Comment (cafemoloko) [reply source/pages/faq.rst:684]: @Erol444 yes, that is our _source of truth_ about HW :)

File: source/pages/troubleshooting.rst [hunk 2]
```diff
@@ -139,13 +139,17 @@ CTRL-C Is Not Stopping It!
...
-Is Your Raspberry Pi Locking Up?
-################################
+Is Your Raspberry Pi Locking Up or is DepthAI Crashing on Raspberry Pi?
+#######################################################################
 
 The Raspberry Pi has a max limit of 1.2A across all its USB ports, and DepthAI/megaAI/OAK can take up to 1A (at max power, usually closer to 500mA).
 
 So if you are seeing lockups, it could be that you are over this 1.2A limit as a result of the total power of the USB devices drawing from the Pi.  Using a powered hub can prevent this, or powering fewer other things off of the Pi over USB.
 
+This may also manifest in DepthAI randomly crashing on the Pi.  This can become particularly often if DepthAI is configured to do many things at once.  This is becoming increasingly likely as we enable DepthAI to do more and more things at once - and thereby increase the maximum power that DepthAI can pull.  It is seeming like the peak power power (current) spikes from DepthAI can go over what the Pi can handle, causing DepthAI to brown-out and return an error.
```
Comment (VanDavv) [inline source/pages/troubleshooting.rst:149]: `peak power power (current)` (typo)

**Pull request merged**